.github/workflows/install_deps/action.yml

```diff
@@ -63,6 +73,13 @@ runs:
         pip install -r requirements-ci.txt --upgrade
         pip install -r requirements-dev.txt --upgrade
 
+    - name: Install numpy 2.0 version
+      shell: bash
+      if: inputs.install_numpy_2 == true
```

Comment:
Does this validate the `default` argument when not specified?

Reply:
@mlxd I am not sure I understand the question. Are you asking what happens to line 84 (that is, to `NUMPY2_VERSION)` if `numpy_2_version` is not specified or `inputs.install_numpy_2 ` is false?